asmcli/components/ca/managed-cas.sh

```diff
@@ -75,6 +75,54 @@ x_enable_workload_certificate_on_membership() {
      -d "${ENABLEFEATURE}" "https://${GKEHUB_API}/v1alpha/projects/${FLEET_ID}/locations/global/features/workloadcertificate?update_mask=membership_specs"
 }
 
+x_wait_for_enabling_workload_certificates() {
+  local GKEHUB_API; GKEHUB_API="$1"
+  local FLEET_ID; FLEET_ID="$2"
+
+  info "Waiting for the workload certificates enablement. This may take up to 20 minutes ..."
+  # Enabling workload certificate feature on a cluster will result in
+  # the cluster restarting and the cluster becoming unreachable for minutes.
+  # The cluster restart may happen at a random time after the enablement.
+  # Therefore, wait 15 minutes before checking the connection to k8s is recovered.
+  sleep 900
+  # Check the connection to k8s is recovered.
+  local REACHABLE; REACHABLE=false
```

Comment:
"false" and "true" are just strings in shell, you should use 0 or 1.

Reply:
Done.